e2e/vmss.go

```diff
@@ -242,3 +249,26 @@ func getBaseVMSSModel(name, location, mcResourceGroupName, subnetID, sshPublicKe
 		},
 	}
 }
+
+func vmssModelHasProperties(vmss *armcompute.VirtualMachineScaleSet) bool {
```

Comment:
hasProperties, hasVMProfile, hasVMNetworkProfile? The parameter already gives a hint that it's a vmss model.

Reply:
might just remove the helpers altogether